alibi_detect/cd/pytorch/context_aware.py

```diff
@@ -22,8 +45,8 @@ def __init__(
             preprocess_x_ref: bool = True,
             update_ref: Optional[Dict[str, int]] = None,
             preprocess_fn: Optional[Callable] = None,
-            x_kernel: Callable = GaussianRBF,
-            c_kernel: Callable = GaussianRBF,
+            x_kernel: BaseKernel = GaussianRBF(init_fn_sigma=_sigma_median_diag),
```

Comment:
Init'ing `GaussianRBF` with kwargs via the defaults here does make deserialization a little more cumbersome...

Reply:
Yes ideally we can do the same across all detectors.